roles/confluent.control-center/templates/control-center_kerberos.properties.j2

```diff
@@ -0,0 +1,13 @@
+# Maintained by Ansible
+{% include './includes/base_control-center.properties.j2' %}
+
+confluent.controlcenter.streams.sasl.mechanism=GSSAPI
+confluent.controlcenter.streams.security.protocol=SASL_PLAINTEXT
+confluent.controlcenter.streams.sasl.kerberos.service.name=kafka
+
+
+confluent.controlcenter.streams.sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required \
+   useKeyTab=true \
+   storeKey=true \
+   keyTab="{{confluent.control.center.keytab}}/controlcenter-{{inventory_hostname}}.keytab" \
```

Comment:
this keytab var is maintained many places... should be one spot

Reply:
Please file a bug for this internally, can be addressed next sprint.

Reply:
I would file a feature request for this.